docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.8"
+
+services:
+  app:
+    image: "periodum_app:${TAG}"
+    container_name: periodum_app_dev
+    build: .
+    volumes:
+      - ./:/usr/src/app # dislike to mount the whole volume
```

Comment:
Burayı _hot reload_'un çalışması için tüm projeyi mount etmem gerekti.

Reply:
Bunda bir sıkıntı var mı şu anda? Mergelenmemiş bu @nesimtunc?